### PR TITLE
PE-45/Warn about user switching focus

### PR DESCRIPTION
**Pull request by jdaev**
Shows appropriate warning message based on wallet type.

### CODE REVIEW

File: lib/app_shell.dart [hunk 1]
```diff
@@ -90,8 +90,27 @@ class _AppShellState extends State<AppShell> {
...
+                                builder: (BuildContext context,
+                                    AsyncSnapshot snapshot) {
+                                  if (snapshot.data ?? false) {
+                                    return ProgressDialog(
+                                      title:
+                                          'Syncing... Please remain on this tab.',
+                                    );
+                                  } else {
+                                    return ProgressDialog(
+                                      title: 'Syncing, please wait',
```
Comment (arielmelendez) [inline lib/app_shell.dart:108]: Let's make the prompts consistent (I know this isn't in the requirements, but it should be). Let's make the copy here:
"Syncing... Please wait."

File: lib/pages/profile_auth/components/profile_auth_loading_screen.dart [hunk 2]
```diff
@@ -4,12 +4,29 @@ import 'package:flutter/material.dart';
...
-        content: const Center(child: CircularProgressIndicator()),
+        content: Column(
+          crossAxisAlignment: CrossAxisAlignment.center,
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: [
+            if (isArConnect)
+              Padding(
+                padding: const EdgeInsets.all(8.0),
+                child: const Text(
+                  'Loading, please remain on this tab.',
```
Comment (arielmelendez) [inline lib/pages/profile_auth/components/profile_auth_loading_screen.dart:25]: The copy from the requirements is:
"Loading... Please remain on this tab."

**Review by arielmelendez — CHANGES_REQUESTED** [review]
A few copy updates need to be made.

File: lib/pages/profile_auth/components/profile_auth_loading_screen.dart (hunk 2 above)
Comment (jdaev) [reply lib/pages/profile_auth/components/profile_auth_loading_screen.dart]: [Corrected ](https://github.com/ardriveapp/ardrive-web/pull/164/commits/c9297c9d6e118e2983c398d49b53c35aff59a624)

File: lib/app_shell.dart (hunk 1 above)
Comment (jdaev) [reply lib/app_shell.dart]: [Corrected ](https://github.com/ardriveapp/ardrive-web/pull/164/commits/c9297c9d6e118e2983c398d49b53c35aff59a624)

**Review by arielmelendez — APPROVED** [review]

**Comment by agsuy** [comment]
Wondering if we could reuse isCurrentProfileArConnect() somewhere as I am not sure where/how we are checking for ArConnect presence on the app (any doc on this?). Besides this I would start testing this right away.

**Review by arielmelendez — DISMISSED** [review]

**Review by arielmelendez — DISMISSED** [review]

**Review by agsuy — APPROVED** [review]
We now emit syncs to console.

Approved.

**Pull request merged**